common/GNUmakefile

```diff
@@ -125,6 +129,26 @@ ifeq ($(TARGET),uefi-aarch64)
         -DUEFI
 endif
 
+ifeq ($(TARGET),uefi-riscv64)
+	ifeq ($(CC_FOR_TARGET_IS_CLANG),yes)
+		override RISCV_CFLAGS += -march=rv64imac -mabi=lp64
+	else
+		override RISCV_CFLAGS += -march=rv64imac_zicsr_zifencei -mabi=lp64
```

Comment:
I'm not sure where the fault lies, but building with this ABI on gcc 11.3 as packaged for https://packages.ubuntu.com/jammy/gcc-riscv64-linux-gnu is getting me the "soft float" ABI which is incompatible with the "double-float" ABI that mintsuki's libgcc binaries were built for, and binutils ld 2.38 is refusing to link the two halves.

As changing it on this end was easier, I switched to an march/mabi with `d` support and got a working build out of it, but I'm not sure who's wrong here.

Reply:
ABI for the libgcc builds was changed in https://github.com/mintsuki/libgcc-binaries/commit/8398d4287aacee886959669e284ca4bf8aad5fb7